service/controller/resource/prometheus/resource.go

```diff
@@ -118,6 +119,15 @@ func toPrometheus(v interface{}, createPVC bool, storageSize resource.Quantity)
 				key.ClusterIDKey(): key.ClusterID(cluster),
 				"cluster_type":     key.ClusterType(cluster),
 			},
+			PodMetadata: &promv1.EmbeddedObjectMetadata{
+				Labels: map[string]string{
+					"giantswarm.io/monitoring":     "true",
+					"app.kubernetes.io/name":       "prometheus",
+					"app.kubernetes.io/managed-by": project.Name(),
+					"app.kubernetes.io/version":    project.Version(),
```

Comment:
This isn't going to be version of _"prometheus"_. I'd skip this label here as it could be confusing. There's also a version field in spec that specifies Prometheus version.